src/compile_offline_NoahMP.sh

```diff
@@ -48,11 +48,11 @@ cd ../..
 ln -sf Land_models/NoahMP LandModel
 cat macros LandModel/hydro/user_build_options.bak  > LandModel/user_build_options
 ln -sf CPL/NoahMP_cpl LandModel_cpl
-make clean; rm -f Run/wrf_hydro_NoahMP.exe ; rm -f Run/*TBL ; rm -f Run/*namelist*
+# make clean; rm -f Run/wrf_hydro_NoahMP.exe ; rm -f Run/*TBL ; rm -f Run/*namelist*
 
 #for debugging and testing
 #make debug; make install; make test
-make && make install
+make -j4 && make install
```

Comment:
Does this work? I had a in-progress PR (#686) to add support for this because it didn't work correctly, but it was never merged.

Reply:
Oh, I didn't actually mean to include that, I'll remove the parallel build because while it did work it was in a docker container which shouldn't have had access to four processors to properly test this

Reply:
In fact I didn't actually mean to commit my local edits to this file, I'll revert them all